### DISCUSSION TITLE
How do I screenshot a region of the screen?

### DISCUSSION THREAD

**Discussion by LeonardoMor** [prasanthrangan/hyprdots#905] [Q&A] [answered]
What the title says.

**Comment by se7en-x230** [comment]
~/.config/hypr/keybindnings.conf

![image](https://github.com/prasanthrangan/hyprdots/assets/69777713/8fb6cf00-857c-47a7-825e-0566b94be8d2)

**Comment by kRHYME7** [comment, marked as answer]
run ``` Hyprdots show binds ``` to see the juicy keybindings/hotkeys.

**Reply by LeonardoMor** [reply]
Is there a bind to show the binds?

**Reply by kRHYME7** [reply]
You can add that yourself. See Hyrprland wiki on how to add keybindings
